IOHexperimenter/Cpp.md

```diff
@@ -72,52 +76,81 @@ const auto &suite_factory = ioh::suite::SuiteRegistry<ioh::problem::Real>::insta
 const auto bbob = suite_factory.create("BBOB", {1, 2}, {1, 3}, {5, 10});
 ```
 
-In the given example, a random search is applied across all problems within a loop :
+In the example, a random search is applied across all the problems within a loop :
 ```cpp
 for (const auto &problem : *bbob) :
 {
   ...
 }
 ```
-The variable `problem` is a pointer of __Problem__ class upper introduced.
+Usage of the `problem` class is introduced [above](#using-individual-problems).
 
 
 <a name = "using-logger"></a>
 ## Using Logger
 
-A default logger is provided to record evaluation information during the optimization process into csv files. The following statement declares a logger, which outputs files into a folder named as `logger_example` at the directory `./`. The algorithm name `random_search` and the algorithm information `a random search for ...` will be attached into the csv files.
+The default logger records evaluation information during the optimization process into csv files. An example of using the default logger is provided in [logger_example.h](https://github.com/IOHprofiler/IOHexperimenter/blob/master/example/logger_example.h)
```

Comment:
It is not exactly csv, but tab-seperated at the moment. Maybe reference the format page here.